pkgs/development/python-modules/kiss-headers/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, buildPythonPackage, fetchFromGitHub, python }:
+
+buildPythonPackage rec {
+  pname = "kiss-headers";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+    owner = "Ousret";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-/eTRyxFyAKQMzE/JjdoEN3w0lRiaIJcsJHTWV8M0CYQ=";
+  };
+
+  checkPhase = ''
+    ${python.interpreter} tests/test_attributes.py
+    ${python.interpreter} tests/test_builder.py
+    ${python.interpreter} tests/test_builder_create.py
+    ${python.interpreter} tests/test_case_insensible_dict.py
+    ${python.interpreter} tests/test_explain.py
+    ${python.interpreter} tests/test_header_operation.py
+    ${python.interpreter} tests/test_header_order.py
+    ${python.interpreter} tests/test_headers.py
+    ${python.interpreter} tests/test_headers_from_string.py
+    ${python.interpreter} tests/test_headers_operation.py
+    ${python.interpreter} tests/test_headers_reserved_keyword.py
+    ${python.interpreter} tests/test_polymorphic.py
```

Comment:
How about we do ``for file in tests/test_*.py``?

Reply:
Wouldn't this consider all the tests? The idea is ignoring `test_serializer.py` and `test_with_http_request.py` that will try to contact an external URL

Reply:
Added a disabledTestPaths section for three of the files failing.